src/python/doc/perslay.rst

```diff
@@ -0,0 +1,65 @@
+:orphan:
+
+.. To get rid of WARNING: document isn't included in any toctree
+
+==============
+PersLay manual
+==============
+
+.. include:: perslay_sum.inc
+
+This module, originally available at https://github.com/MathieuCarriere/perslay, aims at bridging the gap between persistence diagrams and deep learning, by providing the implementation in tensorflow of a neural network layer for persistence diagrams, called PersLay, defined in https://arxiv.org/abs/1904.09378. More specifically, it provides code for reproducing most of the persistence diagram representations in the literature, and optimize their parameters with a neural network.
```

Comment:
https://arxiv.org/abs/1904.09378
Maybe a new entry in `biblio/bibliography.bib` and a `:cite:` tag in the documentation would have a better documentation rendering.